windows/ReactTestApp/MainPage.cpp

```diff
@@ -5,22 +5,16 @@
 #include "ComponentViewModel.h"
 #include "MainPage.g.cpp"
 #include "Manifest.h"
-#include "filesystem"
 
-using namespace winrt::Microsoft::ReactNative;
+using namespace ReactTestApp;
```

Comment:
This seems unused. The only reason you would add this is so you can type `Manifest` instead of `ReactTestApp::Manifest`. Since we're using the fully qualified name below, you don't need this line.

Reply:
It's needed for `JSBundleSource`, but can use fully qualified name there sa well.